modules/exploits/linux/http/tr064_ntpserver_cmdinject.rb

```diff
@@ -104,6 +104,7 @@ def execute_command(cmd, opts)
   def check
     begin
       res = send_request_cgi({
+'method' => 'GET',
```

Comment:
yes, please honor existing whitespace conventions for Ruby (2 space characters, indented from the previous line since this is a continuation).